Add more clarity on the middleware configuration (#107979)
Azure/azure-content: articles/active-directory/develop/scenario-web-api-call-api-app-configuration.md

builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
    .AddMicrosoftIdentityWebApi(Configuration, "AzureAd")
    .EnableTokenAcquisitionToCallDownstreamApi()
    .AddDownstreamWebApi("MyApi", Configuration.GetSection("MyApiScope"))
    .AddInMemoryTokenCaches();
// ...
```

> If the web app needs to call another APIAPI resource, you could repeat the `.AddDownstreamWebApi()` method with the relavent scope as below.

Reviewer: ```suggestion
If the web app needs to call another API resource, repeat the `.AddDownstreamWebApi()` method with the relevant scope as shown in the following snippet:
```

```csharp
using Microsoft.Identity.Web;

// ...
builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
    .AddMicrosoftIdentityWebApi(Configuration, "AzureAd")
    .EnableTokenAcquisitionToCallDownstreamApi()
    .AddDownstreamWebApi("MyApi", Configuration.GetSection("MyApiScope"))
    .AddDownstreamWebApi("MyApi2", Configuration.GetSection("MyApi2Scope"))
    .AddInMemoryTokenCaches();
// ...
```

> Note that`.EnableTokenAcquisitionToCallDownstreamApi` is called without any parameter, which means that the access token will be acquired just in time as the controller requests the token by specifying the scope.  

Reviewer: ```suggestion
Note that `.EnableTokenAcquisitionToCallDownstreamApi` is called without any parameter, which means that the access token will be acquired just in time as the controller requests the token by specifying the scope.  
```

> Please do note that, in the above code snippets `.EnableTokenAcquisitionToCallDownstreamApi` is called without any parameter which means, access token will be acquired just in time as the controller requests the token by specifying the scope.  

Reviewer: ```suggestion
Note that`.EnableTokenAcquisitionToCallDownstreamApi` is called without any parameter, which means that the access token will be acquired just in time as the controller requests the token by specifying the scope.  
```

> The scope can also be passed in when calling the `.EnableTokenAcquisitionToCallDownstreamApi` which would make the web app to acquire the token during the initial user login itself and then the token will be served from the cache when controller requests it.  

Reviewer: ```suggestion
The scope can also be passed in when calling `.EnableTokenAcquisitionToCallDownstreamApi`, which would make the web app acquire the token during the initial user login itself. The token will then be pulled from the cache when controller requests it.  
```

> You could also pass-in the scope when calling the `.EnableTokenAcquisitionToCallDownstreamApi` which would make the web app to acquire the token during the initial user login itself and then the token will be served from the cache when controller requests it.  

Reviewer: ```suggestion
The scope can also be passed in when calling the `.EnableTokenAcquisitionToCallDownstreamApi` which would make the web app to acquire the token during the initial user login itself and then the token will be served from the cache when controller requests it.  
```
